bin/start-user-internet-available

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+# start-internet-available-user
+#
+# Copyright (C) 2019 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Script that gets executed as part of the system service
+# start-internet-available-user.service. It will start
+# internet-available-user.target for all currently logged in users.
+#
+# Requires sudo to run.
+
+
+import os
+import sys
+
+if __name__ == '__main__' and __package__ is None:
+    dir_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'src'))
+    if dir_path != '/usr':
+        sys.path.insert(1, dir_path)
```

Comment:
We are using this code snippet in numerous places and it still confuses me.
What does it do? Can we add a clarifying comment around it?

Reply:
It's main purpose is to extend the `sys.path` with the local directory of the repo such that subsequent import statements prefer the local version rather than the system version of installed packages